Gemfile.lock

```diff
@@ -71,73 +71,73 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (2.0.0)
       rails (>= 7.0)
-    activejob (7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activejob (7.0.4)
+      activesupport (= 7.0.4)
       globalid (>= 0.3.6)
-    activemodel (7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activerecord (7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activestorage (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activemodel (7.0.4)
+      activesupport (= 7.0.4)
+    activerecord (7.0.4)
+      activemodel (= 7.0.4)
+      activesupport (= 7.0.4)
+    activestorage (7.0.4)
+      actionpack (= 7.0.4)
+      activejob (= 7.0.4)
+      activerecord (= 7.0.4)
+      activesupport (= 7.0.4)
       marcel (~> 1.0)
       mini_mime (>= 1.1.0)
-    activesupport (7.0.2.3)
+    activesupport (7.0.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
-    acts_as_tenant (0.5.1)
+    acts_as_tenant (0.5.2)
       rails (>= 5.2)
       request_store (>= 1.0.5)
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
-    airbrussh (1.4.0)
+    addressable (2.8.1)
+      public_suffix (>= 2.0.2, < 6.0)
+    airbrussh (1.4.1)
```

Comment:
### [airbrussh](https://github.com/mattbrictson/airbrussh)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.4.0 → 1.4.1

[_[change-log](https://github.com/mattbrictson/airbrussh/releases), [source-code](https://github.com/mattbrictson/airbrussh), [gem-diff](https://my.diffend.io/gems/airbrussh/1.4.0/1.4.1)_]





<details>
<summary>Commits</summary>

A change of **10** commits. See the full changes on [the compare page](https://github.com/mattbrictson/airbrussh/compare/v1.4.0...v1.4.1).

These are the individual commits:
- (ccba560) [Fix travis build failure with ruby-head (#132)](https://github.com/mattbrictson/airbrussh/commit/ccba560d26ac17b8e0a77f7c499739a91b738ce1)
- (3a2c6b6) [Happy New Year! 🥳](https://github.com/mattbrictson/airbrussh/commit/3a2c6b651333bc7cd05ea77ec91d30855cba8e88)
- (adea7ff) [Replace Travis with CircleCI and add Ruby 3.0 and 3.1 to CI …](https://github.com/mattbrictson/airbrussh/commit/adea7ff5921b04f2a5149abcfca7222ee91b0a4d)
- (a29b63a) [Simplify release-drafter config (#139)](https://github.com/mattbrictson/airbrussh/commit/a29b63affd5ea4c532191ff885a2e4e523b2e2b6)
- (b2223e2) [Rename default branch to main (#140)](https://github.com/mattbrictson/airbrussh/commit/b2223e232ce3ebfbe57112688e7f7659e2c009d6)
- (0ae0026) [Pin docile so that builds work with Ruby < 2.5 (#141)](https://github.com/mattbrictson/airbrussh/commit/0ae002665df9fd2c5119444249b1367e8a5d88d4)
- (a6d5063) [Update appveyor to use a more modern version of Ruby (2.6) (…](https://github.com/mattbrictson/airbrussh/commit/a6d506317b07f210991b6081159fbc97579431c3)
- (0d26313) [Upgrade to modern version of coveralls gem (#143)](https://github.com/mattbrictson/airbrussh/commit/0d26313dd3eab1e1b94aca893dde976ff9c6ef51)
- (44b95a0) [Include link to full diff in generated release notes (#144)](https://github.com/mattbrictson/airbrussh/commit/44b95a09f9479505cfcff61a23fc213e5e647eea)
- (d440931) [Release 1.4.1](https://github.com/mattbrictson/airbrussh/commit/d440931db4f593fd0dbf14e310d6173f9b6973d3)


</details>